modules/apps/repository/repository-test/src/testIntegration/java/com/liferay/repository/test/RepositoryFactoryTest.java

```diff
@@ -31,29 +55,80 @@ public class RepositoryFactoryTest {
 	@ClassRule
 	@Rule
 	public static final AggregateTestRule aggregateTestRule =
-		new LiferayIntegrationTestRule();
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
 
 	@Before
 	public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testAddFolderInRepositoryWhenSameRepositoryIdExistsInDifferentPartition()
+		throws Exception {
+
+		if (!DBPartition.isPartitionEnabled()) {
+			return;
```

Comment:
Will the CI return true to DBPartition.isPartitionEnabled() by default?

If not this test will not execute.

I've agreed with @marcogalluzzi in slack to confirm this with CI/test admins

Reply:
Good to go except the previous comment. 👍 

@marcogalluzzi if you confirm that the test will be run, please forward the pull.

Reply:
You should probably use something like `Assume.assumeTrue()` here.